linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_sysadmin_actions/ansible/shared.yml

```diff
@@ -0,0 +1,43 @@
+# platform = multi_platform_rhel
+# reboot = false
+# complexity = low
+
+- name: Search /etc/audit/rules.d for other sudoers audit rules
+  find:
+    paths: /etc/audit/rules.d
+    recurse: false
+    contains: -k sudoers$
```

Comment:
Rule is about key `action` in path `/etc/sudoers`, not about key `sudoers`. Please change all checks to `-k actions`.